duckduckgo/macos-browser
# Fix wrong value for WebsiteBreakage `reportFlow` parameter

Task/Issue URL: https://app.asana.com/0/1201037661562251/1204998650870784/f

**Description**:
During the Privacy Dashboard rework we changed how the website breakage is reported. Instead of gathering data and sending the pixel directly from the JS we send a callback to the native code and reuse already existing website breakage reporting code. The `reportFlow` request parameter is set to a fixed `native` value but it should pass `dashboard` when triggered from the Privacy Dashboard.

**Steps to test this PR**:
Send broken site report from Privacy Dashboard and confirm that `reportFlow` request parameter is set to `dashboard`.
1. Open Privacy Dashboard
2. Click "Website not working as expected?"
3. Send the report

Send broken site report from the menu and confirm that `reportFlow` request parameter is set to `native`.
1. Click "..." -> "Send Feedback"
2. Select "Report a site not working as expected"
3. Send the report
4. Click "Website not working as expected?"
5. Send Report



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
